source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+As it turned out setting up multiple hosts at once can be rather tedious via
+UI. The slow point is that the administrator has to go through every new host
+and do Setup Host Network manually. By utilizing Copy Host Networking, the administrator
+has to set only single host, for the rest the administrator can just specify the host
+to take the network configuration from. 
+
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
+
+The Copy Host Network helps the administrator with Day 1 deployment automation
```

Comment:
English: The Copy Host Network *command* helps

Reply:
I don't know if everyone knows what "Day 1" means. I'd use quotes to make clearer that this is an idiom.

Reply:
Done